demos/common/cpp/utils/include/utils/images_capture.h

```diff
@@ -9,6 +9,9 @@
 #include <opencv2/videoio.hpp>
 #include "utils/performance_metrics.hpp"
 
+constexpr static bool SAFE_READ = true;
+constexpr static bool EFFICIENT_READ = false;
```

Comment:
Would it better to use enum instead? This will allow to specify type for argument and rename  `safe_read ` arg to `read_type` for example, because now it's a bit clumsy:
```
bool safe_read=EFFICIENT_READ
```

Reply:
Added